src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -100,6 +100,10 @@ void SolverInterfaceImpl:: configure
 (
   const std::string& configurationFileName )
 {
+  mesh::Mesh::resetGeometryIDsGlobally();
```

Comment:
@uekerman Can you check your fix?

Reply:
This is a workaround that I suggested to @Dominanz, see #378 
We should exclude this from the PR as it is unrelated to the Matlab bindings.

Reply:
I agree. Does this mean that we do not want to introduce this change at all or just introduce it in a different PR?

Reply:
I removed the code changes from this PR (https://github.com/precice/precice/pull/494/commits/e922868e6258bb35cf095106483e9307fc679c56) and added a comment in the `README.md` file of the bindings (40b16c7).